galeracluster/source/documentation/galera-parameters.rst

```diff
@@ -865,6 +876,22 @@ The excerpt below is an example of how this Galera parameter might look in the c
    wsrep_provider_options="gcache.keep_pages_size=0"
 
 
+.. _`gcache.mem_size`:
+.. rst-class:: section-heading
+.. rubric:: ``gcache.mem_size``
+
+.. index::
+   pair: wsrep Provider Options; gcache.mem_size
+
+The maximum size of size of the ``malloc()`` store for setups that have spare RAM.

```

Comment:
I feel this is a bit complex for people (non-programmers) to understand what malloc() is. I think this can be reworked frankly